### ISSUE TITLE
Support Websocket Streaming API

### ISSUE THREAD

**Issue by zainryan** [marpaia/alpaca-trade-api-cpp#5]
[This feature](https://alpaca.markets/docs/api-documentation/api-v2/streaming/) used to be invited only but now goes public. Any plan to support this?

**Comment by marpaia** [comment]
The functionality that [you linked](https://alpaca.markets/docs/api-documentation/api-v2/streaming/) is currently supported [here](https://github.com/marpaia/alpaca-trade-api-cpp/blob/3f021a45b545d2fcdf480b31f53b099c670e67ed/alpaca/streaming.h). I'm hoping to work on adding support for the new stream types which are documented [here](https://alpaca.markets/docs/api-documentation/api-v2/market-data/streaming/). I hope that helps. Is that what you mean?

**Comment by zainryan** [comment]
Yes, what I mean is the new market streaming API. It would be greatly appreciated if that is supported.

**Comment by marpaia** [comment]
Cool, I'm definitely planning on added support for that soon. Stay tuned!

**Comment by codinghaven** [comment]
Hi, thank you for your work, any updates on this feature, eager to hear, thanks!
